runners-template-vms-creating/isos/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
See above. Please create only one .gitignore in the rootdirectory

Reply:
See above.